static/js/sub_store.ts

```diff
@@ -0,0 +1,68 @@
+import * as blueslip from "./blueslip";
+
+export type StreamSubscription = {
+    audible_notifications: boolean | null;
+    color: string;
+    date_created: number;
+    description: string;
+    desktop_notifications: boolean | null;
+    email_address: string;
+    email_notifications: boolean | null;
+    first_message_id: number | null;
+    history_public_to_subscribers: boolean;
+    in_home_view: boolean;
+    invite_only: boolean;
+    is_announcement_only: boolean;
+    is_muted: boolean;
+    is_web_public: boolean;
+    message_retention_days: number | null;
+    name: string;
+    pin_to_top: boolean;
+    push_notifications: boolean | null;
+    rendered_description: string;
+    role: 20 | 50;
+    stream_id: number;
+    stream_post_policy: 1 | 2 | 3 | 4;
+    stream_weekly_traffic: number | null;
+    subscribers?: number[];
+    wildcard_mentions_notify: boolean;
```

Comment:
Comparing with [zerver/lib/event_schema.py](https://github.com/zulip/zulip/blob/c99c423c81e5d8737011910a47acbce6066d5312/zerver/lib/event_schema.py#L84), this can apparently be `null`.

Reply:
Yep, fixed.